Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -939,7 +939,15 @@ public class PBXProjGenerator {
         }
 
         let swiftObjCInterfaceHeader = project.getCombinedBuildSetting("SWIFT_OBJC_INTERFACE_HEADER_NAME", target: target, config: project.configs[0]) as? String
-        let swiftInstallObjCHeader = project.getCombinedBuildSetting("SWIFT_INSTALL_OBJC_HEADER", target: target, config: project.configs[0]) as? Bool
+        let swiftInstallObjCHeaderValue = project.getCombinedBuildSetting("SWIFT_INSTALL_OBJC_HEADER", target: target, config: project.configs[0])
+
+        var swiftInstallObjCHeader = true // Xcode default
```

Comment:
Now that this is a non optional, can we update the confusing line below
```swift
&& swiftInstallObjCHeader != false
```
to just:
```swift
&& swiftInstallObjCHeader
```

Reply:
Done 👍